src/pycontw2016/settings/base.py

```diff
@@ -333,7 +334,7 @@ def node_bin(name):
     (datetime.date(2024, 9, 22), _('Day 2')),
 ])
 
-COC_VERSION = '2024.0'
+COC_VERSION = '2025.0'
```

Comment:
CoC (code of conduct) hasn't changed or adjusted. You should maintain it to be version "2024".